pengwin-setup.d/terminal.sh

```diff
@@ -0,0 +1,139 @@
+source $(dirname "$0")/common.sh "$@"
+
+function main() {
+
+  local menu_choice=$(
+
+    menu --title "Terminal Menu" --checklist --separate-output "Select the terminals you want to install\n[SPACE to select, ENTER to confirm]:" 14 60 7 \
+      "WINTERM" "Windows Terminal" off \
+      "WSLTTY" "WSLtty" off \
+      "TILIX" "Tilix (requires X Server)" off \
+      "GTERM" "Gnome Terminal (requires X Server)" off \
+      "XFTERM" "Xfce Terminal (requires X Server)" off \
+      "TERMIN" "Terminator (requires X Server)" off \
+      "KONSO" "Konsole (requires X Server)" off \
+
+  3>&1 1>&2 2>&3)
+
+  if [[ ${menu_choice} == "CANCELLED" ]] ; then
+    return 1
+  fi
+
+  if [[ ${menu_choice} == *"WINTERM"* ]] ; then
+    echo "WINTERM"
+    if (confirm --title "Windows Terminal" --yesno "Would you like to install Windows Terminal?" 8 40) ; then
+      tmp_win_version=$(wslsys -B -s)
+      if [ $tmp_win_version -lt 18362 ]; then
+        whiptail --title "Unsupported Windows 10 Build" --msgbox "Windows Terminal requires Windows 10 Build 18362, but you are using $tmp_win_version. Skipping Windows Terminal." 8 56
+        return
+      fi
+
+      if (whiptail --title "Windows Terminal" --yes-button "Store" --no-button "GitHub" --yesno "Would you like to install the store version or GitHub version?" 8 80) then
+        wslview "ms-windows-store://pdp/?ProductId=9n0dx20hk701"
+      else
+        createtmp
+
+        winterminal_url="$(curl -s https://api.github.com/repos/microsoft/terminal/releases | grep 'browser_' | head -1 | cut -d\" -f4)"
```

Comment:
Let's discuss this. I have seen many problems with dependencies in users trying to install it directly instead of using the store

Reply:
Yeah, I believe the dependency problem might will cause the issue... Did find that it is still an issue not fixed: https://github.com/microsoft/terminal/issues/1386. Maybe commented out temporarily until it is resolved?